nifi-nar-bundles/nifi-influxdb-bundle/nifi-influxdb-processors/src/main/java/org/apache/nifi/processors/influxdb/ExecuteInfluxDBQuery.java

```diff
@@ -86,6 +93,18 @@
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .build();
 
+    public static final PropertyDescriptor INFLUX_DB_QUERY_CHUNK_SIZE = new PropertyDescriptor.Builder()
+            .name("influxdb-query-chunk-size")
+            .displayName("Results chunk size")
+            .description("Chunking can be used to return results in a stream of smaller batches "
+                + "(each has a partial results up to a chunk size) rather than as a single response. "
+                + "Chunking queries can return an unlimited number of rows. Note: Chunking is enable when result chunk size is greater than 0")
+            .defaultValue(String.valueOf(DEFAULT_INFLUX_RESPONSE_CHUNK_SIZE))
+            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
+            .addValidator(StandardValidators.createLongValidator(0, Integer.MAX_VALUE, true))
```

Comment:
If the default query result limit is 10k, what is the likelihood that there is a good reason for this to be above 10k? Even if we go beyond that for chunks, Integer.MAX_VALUE is way past a sane limit.

Reply:
According to InfluxDB [documentation](https://docs.influxdata.com/influxdb/v1.5/guides/querying_data/).  
> The maximum row limit only applies to non-chunked queries. Chunked queries can return an unlimited number of points.  

How big limit do you suggest ?

Reply:
If the default limit is 10k for the server, I think that should be taken as a warning sign that you really need to know what you're doing if you mess around beyond that. So let's just go with 10k for now.

Reply:
@timhallinflux Can you jump in here and give us your opinion?

Reply:
Aligning with the default value seems very rational.

Reply:
Ok, go with the default @michalmisiewicz

Reply:
@michalmisiewicz I'll just go ahead and do the fix and merge. Thanks for the patch.